build/lib/pgBackRestBuild/Config/Data.pm

```diff
@@ -298,6 +298,7 @@ use constant CFGOPT_PG_HOST_CONFIG_PATH                             => CFGOPT_PG
 use constant CFGOPT_PG_HOST_PORT                                    => CFGOPT_PG_HOST . '-port';
 use constant CFGOPT_PG_HOST_USER                                    => CFGOPT_PG_HOST . '-user';
 
+use constant CFGOPT_PG_DATABASE                                     => CFGDEF_PREFIX_PG . '-database';
```

Comment:
Not sure where to put this comment, but something else must be required in the configuration or the code. I checked out dev-dbname, created the docs and ran the check command:
```
postgres@pg-primary:~$ pgbackrest --stanza=demo --pg-database=silly check
ERROR: [031]: invalid option '--pg-database=silly'
```
Running help confirms the option is listed but not one of the commands gets passed the error.

Reply:
Don't you mean `--pg1-database=silly`?

Reply:
Ah, yes, it's indexed and I forgot the index. Works fine when I use with the index.